lms/djangoapps/mobile_api/models.py

```diff
@@ -25,3 +23,51 @@ def get_video_profiles(cls):
         Get the list of profiles in priority order when requesting from VAL
         """
         return [profile.strip() for profile in cls.current().video_profiles.split(",") if profile]
+
+
+class AppVersionConfig(ConfigurationModel):  # pylint: disable=model-missing-unicode
```

Comment:
Instead of suppressing the pylint issue, you should add a `__unicode__()` method instead.